CalculatorTests/Tests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using NUnit.Framework;
+using static Calculator.Program;
+
+namespace CalculatorTests
+{
+    [TestFixture]
+    public class Tests
+    {
+        [Test]
+        public void AddOneNumber()
+        {
+            int answer = CalculateAnswer("+", new[] {5});
+            Assert.AreEqual(5, answer);
+        }
+
+        [Test]
+        public void AddTwoNumbers()
+        {
+            int answer = CalculateAnswer("+", new[] {5, 7});
+            Assert.AreEqual(12, answer);
+        }
+
+        [Test]
+        public void AddManyNumbers()
+        {
+            int answer = CalculateAnswer("+", new[] {1, 2, 3, 4, 5, 6});
+            Assert.AreEqual(21, answer);
+        }
+
+        [Test]
+        public void ThrowException()
+        {
+            Assert.Throws<IndexOutOfRangeException>(() => CalculateAnswer("+", new int[] {}));
+        }
```

Comment:
It would be nice to know why this throws an exception. Perhaps renaming the method to `ThrowsException_WhenArgumentListIstEmpty` might be better.

Also, having read your implementation of this method, this depends on the runtime throwing an exception when you make an invalid access to the non-existent first element. I think it would be better if this was explicitly checked and thrown by you in `CalculateAnswer`, perhaps with a custom exception class accurately naming the error.